lib/improver/tests/acceptance/test_nowcast_optical_flow.py

```diff
@@ -69,10 +69,10 @@ def test_metadata(tmp_path):
     input_paths = [kgo_dir / f"20181103{hhmm}_{RADAR_REGRID}.nc"
                    for hhmm in ("1530", "1545", "1600")]
     oe_path = kgo_dir / OE
-    # TODO: the BATS test does not call improver with the metadata file
-    # metadata_path = kgo_dir / "../metadata/precip.json"
+    metadata_path = kgo_dir / "../metadata/precip.json"
     output_path = tmp_path / "output.nc"
     args = [oe_path, *input_paths,
+            "--attributes-dict", metadata_path,
```

Comment:
Looking at the code, optical flow should not do this in the first place (now that it has no "extrapolate" option, and definitely shouldn't use the precipitation nowcast metadata.  Two options here: you could:
- Remove this command line option and update KGO for that, or
- Don't make this change as I will be addressing it anyway in IMPRO-1491

Reply:
I will go with the latter.